src/ssim.jl

```diff
@@ -111,6 +111,11 @@ function _ssim_map(iqi::SSIM, x::GenericGrayImage, ref::GenericGrayImage, peakva
         c = @. (2σx_σy + C₂)/(σx² + σy² + C₂) # equation (9) in [1]
         s = @. (σxy + C₃)/(σx_σy + C₃) # equation (10) in [1]
 
+        # ensure that negative numbers in s are not being raised to powers less
+        # than 1, non-standard implementation
+        if γ < 1.0
+            s = max.(s, zero(eltype(s)))
```

Comment:
In Matlab R2020a, it force a `s = abs.(s)` kind of change for all cases, including the default cases when `[α, β, γ] ≈ [1.0, 1.0, 1.0]`

FYI, here's the reply from MATLAB developers wrt the SSIM changes in R2020a:

> 1. MATLAB implemented the algo exactly like the ref paper
> 
> Refer to
> https://www.mathworks.com/help/images/ref/ssim.html
> References
> 
> 2. For some combinations of input, this results in complex valued outputs. In this context, complex valued output does not make sense.
> 
> 3. Based on our understanding of the algo, we think taking the abs value of the covariance is the right thing to do in this context.
>
> So, results in 20a is "correct". However please consider, if the complex valued output make sense for you.

I'm trying to get involved and "persuade" them that this isn't correct.